packages/react/toast/src/Toast.tsx

```diff
@@ -574,10 +572,6 @@ const ToastImpl = React.forwardRef<ToastImplElement, ToastImplProps>(
                 })}
               >
                 <Primitive.li
-                  // Ensure toasts are announced as status list or status when focused
-                  role="status"
```

Comment:
my memory on this is flaky but the comment above seems to imply i intentionally added this to ensure "status list" was read aloud. 

i recall purposefully using `<li>`s for this to ensure it communicated the number of toasts in the toasts list too so @benoitgrelard @andy-hook it would be good to double-check none of that is broken with these changes.